src/python/pants/base/build_root.py

```diff
@@ -46,7 +46,8 @@ def pathlib_path(self) -> Path:
     def path(self) -> str:
         """Returns the build root for the current workspace."""
         if self._root_dir is None:
-            # This env variable is for testing purpose.
+            # Do not remove/change this env var without coordinating with `pantsbuild/scie-pants` as
+            # it is being used when running Pants from sources on a repo.
```

Comment:
This part is not true, its being used in all invocations because of legacy oddness. The current sentinels are `pants`, `BUILD_ROOT` and `BUILDROOT` but _not_ `pants.toml`. In a repo without `pants` checked in this means scie-pants needs to get sneaky.

Reply:
wouldn't it make sense for Pants to look for `pants.toml` rather than `pants` now (or both), going forward?

Reply:
I went ahead and added it to the list of sentinel files.. not that it changes much for `scie-pants` right away, but in a few years time perhaps...